mw-webapp/src/component/header/HeaderContext.ts

```diff
@@ -0,0 +1,32 @@
+import {createContext, useContext} from "react";
+import {User} from "firebase/auth";
+
+const STUB_USER = null;
+
+/**
+ * Stub for set user
+ */
+const STUB_SET_USER = () => undefined;
```

Comment:
why does it set undefined if user type = User | null

Reply:
Deleted